ts/src/test/test.ts

```diff
@@ -133,7 +133,7 @@ export default class testMainClass extends baseMainTestClass {
     async init (exchangeId, symbol) {
         this.parseCliArgs ();
         const symbolStr = symbol !== undefined ? symbol : 'all';
-        console.log ('\nTESTING ', ext, { 'exchange': exchangeId, 'symbol': symbolStr }, '\n');
+        dump ('\nTESTING ', ext, { 'exchange': exchangeId, 'symbol': symbolStr }, '\n');
```

Comment:
this was leftover, we nowhere use console.log, instead we everywhere used `dump`, which outputs much better in `php/py`